python/python/glide/async_commands/core.py

```diff
@@ -1324,6 +1428,124 @@ async def zcount(
             ),
         )
 
+    async def zrange(
+        self,
+        key: str,
+        range_query: Union[RangeByIndex, RangeByLex, RangeByScore],
+        reverse: bool = False,
+    ) -> List[str]:
+        """
+        Returns the specified range of elements in the sorted set stored at `key`.
+
+        ZRANGE can perform different types of range queries: by index (rank), by the score, or by lexicographical order.
+
+        See https://redis.io/commands/zrange/ for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+            range_query (Union[RangeByIndex, RangeByLex, RangeByScore]): The range query object representing the type of range query to perform.
+                - For range queries by index (rank), use RangeByIndex.
+                - For range queries by lexicographical order, use RangeByLex.
+                - For range queries by score, use RangeByScore.
+            reverse (bool): If True, reverses the sorted set, with index 0 as the element with the highest score
+                By default, `start` must be less than or equal to `stop` to return anything.
```

Comment:
This shouldn't be under 'args' documentation, and I think it should be here at all. Please add to each of the ranges the required note (e.g. for index range documentation add: 
_"`start` must be less than or equal to `stop` to return anything."_,
 but for RangeByScore write
  "_the `start` is the highest score to consider, and `stop` is the lowest score to consider,
                therefore `start` must be greater than or equal to `stop` in order to return anything._"

Reply:
BTW, from your example - stop can be less than start in RangeByIndex (RangeByIndex(0, -1)), so this note isn't accurate. think if you need to add it at all